### PR TITLE
remove jcenter

### PR DESCRIPTION
**Pull request by epicadk**
Thanks to [Coolmind](https://stackoverflow.com/questions/49573157/android-studio-3-1-could-not-find-org-jetbrains-trove4jtrove4j20160824)

### CODE REVIEW

**Comment by epicadk** [comment]
Not exactly sure why the upload is failing. I will look into it.

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -47,5 +47,10 @@
       <option name="name" value="maven" />
       <option name="url" value="https://oss.sonatype.org/content/repositories/snapshots" />
     </remote-repository>
+    <remote-repository>
+      <option name="id" value="Gradle Central Plugin Repository" />
+      <option name="name" value="Gradle Central Plugin Repository" />
+      <option name="url" value="https://plugins.gradle.org/m2" />
+    </remote-repository>
```
Comment (jingtang10) [inline .idea/jarRepositories.xml:54]: do you actually need this?

File: build.gradle [hunk 2]
```diff
@@ -3,7 +3,8 @@ buildscript {
     apply from: 'deps.gradle'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
+        gradlePluginPortal()
```
Comment (jingtang10) [inline build.gradle:7]: have you tested without this? which artifact do you need this for?

File: build.gradle [hunk 3]
```diff
@@ -16,8 +17,8 @@ buildscript {
 allprojects {
     repositories {
         google()
-        jcenter()
         mavenCentral()
+        gradlePluginPortal()
```
Comment (jingtang10) [inline build.gradle:21]: and this?

**Comment by jingtang10** [comment]
Thanks for the fix! A few comments.

**Review by jingtang10 — CHANGES_REQUESTED** [review]
please see comments.

File: build.gradle (hunk 2 above)
Comment (epicadk) [reply build.gradle:7]: It's required for this artifact `org.jetbrains.trove4j:trove4j:20160824`

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (epicadk) [reply .idea/jarRepositories.xml]: Sorry wasn't sure. Removed it.

**Comment by s-ayush2903** [comment]
@epicadk Can you please keep this PR intact for some time, as #364 is almost to finish and if this'll get merged then there'll be a ton of conflicts to resolve as the aforementioned PR has so many changes :pleading_face:

File: build.gradle (hunk 3 above)
Comment (epicadk) [reply build.gradle:21]: same as above.

**Review by jingtang10 — APPROVED** [review]
Thanks for this fix! 👍

**Pull request merged**